styles/styles.css

```diff
@@ -333,6 +360,65 @@ form button[type="submit"]:hover {
   color: var(--body-color);
 }
 
+.mb-1 {
```

Comment:
I understand the motivation behind this, but I'm not a fan - from here to line 421. 

I don't think this approach will add a benefit, and may actually create more issues - as you have to know what the values of the classes mean, and add them as you need.

Whereas in a block's css we can be specific and targeted with the configuration.

Reply:
ok. I will left them for property search block only